.github/dependabot.yaml

```diff
@@ -45,3 +45,19 @@ updates:
       action-deps:
         patterns:
           - '*'
+  - package-ecosystem: github-actions
+    directory: .github/actions/install-grafana
```

Comment:
If this is intended for Pyroscope, should this action be for installing pyroscope or be for install Grafana?